metadata-load/src/test/resources/application.properties

```diff
@@ -15,8 +15,9 @@ metadata.datasource.driver-class-name=org.h2.Driver
 spring.main.web-environment=false
 spring.jmx.enabled=false
 
-# Entrez API querying settings.
-# For testing purposes, using no API key and increasing delay to 1000 to accommodate lower allowed request rate
+# Entrez API key
 entrez.api.key=
-entrez.api.attempts=10
-entrez.api.delay=1000
+
+# Entrez and ENA API retry attempts and request delay (can be as low as 500 ms).
```

Comment:
it is said as delay can be as low as 500 ms, but the delay is 1000 ??

Reply:
For testing purpose delay is kept as 1000 for better accommodation